docs/guides/OneDrive.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Unsync/Unlink/Uninstall OneDrive
+nav_exclude: false
+has_children: false
+parent: General Guides
+search_exclude: false
+last_modified_date: 2022-06-28
+---
+
+# Unsyncing/Uninstalling One Drive from your System
+
+## Unsyncing for a short period of time
+If you want to stop syncing OneDrive on your computer because you are on a metered or slow internet connection, you can Pause and Resume your sync. Pausing and Resuming can also help if OneDrive status displays "Processing changes" for a long time.
+
+### To pause syncing
+1. Select the white or blue OneDrive cloud icon in the notification/menu area. This icon may be under the "show hidden icons"tab on your taskbar. (See pictures for refrencing below)
+![Show Hidden Icons](https://imgur.com/9Eibny8.png) 
+![One Drive Icon](https://imgur.com/XPr89uv.png)
+2. Right-click (or CTRL-click in Mac) the OneDrive icon on the taskbar, Click the        ![settings](https://support.content.office.net/en-us/media/9c35458d-dc74-4d37-9cc7-89aff307eff9.png) icon and select Pause syncing > 2 / 8 / 24hours.

```

Comment:
You could upload the images under the assets folder in the main folder. Just remember to make your own folder for the specific article. If you want, you can add sub folders for each OS for your convenience.